catalog/service/impl/src/main/java/org/projectnessie/catalog/service/impl/ImportSnapshotWorker.java

```diff
@@ -178,7 +178,8 @@ private EntitySnapshotObj.Builder importIcebergView(
       try {
 
         InputStream input = taskRequest.objectIO().readObject(metadataLocation);
-        if (metadataLocation.requiredPath().endsWith(".gz")) {
+        if (metadataLocation.requiredPath().endsWith(".gz")
+            || metadataLocation.requiredPath().endsWith(".gz.metadata.json")) {
```

Comment:
nit: maybe refactor common logic wrt lines 97-98 into a `private` method?

Reply:
Done. Also ran `./gradlew spotlessApply` to fix style issues.